docs/layers/column-layer.md

```diff
@@ -172,6 +211,12 @@ The elevation of each cell in meters.
 * If a number is provided, it is used as the elevation for all objects.
 * If a function is provided, it is called on each object to retrieve its elevation.
 
+## Sub Layers
+
+The PolygonLayer renders the following sublayers:
```

Comment:
Remove this. There is no `Sub Layers` in `ColumnLayer`. `Sub Layers` means deck.gl layers.